mysql/toddl.go

```diff
@@ -191,3 +192,31 @@ func cvtPrimaryKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Key)
 	}
 	return spKeys
 }
+
+func cvtForeignKeys(conv *internal.Conv, srcTable string, srcFkeys []schema.Fkey) []ddl.Foreignkey {
+	var spFkeys []ddl.Foreignkey
```

Comment:
`spFKeys`?

Reply:
spanner foreign keys

Reply:
Given that this function only deals with foreign keys, I think we can just drop "F" in the local variables in this function and just refer to them as keys. Note that in contexts where we use both primary keys and foreign keys at the same time, we shouldn't just use raw "key" -- instead it should be pKey and fKey to avoid confusion (or maybe just pk and fk).

How about:
	var spKeys []ddl.Foreignkey

and a few lines later:
        for _, key := range srcFkeys {

and for the function parameter:
         srcKeys []schema.Fkey

Reply:
Sure, got it. Done.